app/models/state_file_base_intake.rb

```diff
@@ -144,7 +144,11 @@ def first_name_and_middle_initial
     end
 
     def has_itin?
-      @ssn.start_with?("9")
+      starts_with_9 = @ssn.start_with?('9')
+      digits_4_and_5 = @ssn[3, 2].to_i
+
+      in_valid_range = (50..65).include?(digits_4_and_5) || (70..88).include?(digits_4_and_5) || (90..92).include?(digits_4_and_5) || (94..99).include?(digits_4_and_5)
+      starts_with_9 && in_valid_range
```

Comment:
Might be good to add a unit test on this method

Reply:
i added a new unit test per this comment in the az140 spec! Let me know if this works!